ddmrp/views/stock_buffer_view.xml

```diff
@@ -54,6 +64,13 @@
                     icon="fa-cogs"
                     type="action"
                 />
+                <button
+                    string="See Long Term Supply"
```

Comment:
I would change to "Some incoming quantities are outside of the Decoupled Lead Time Horizon and may require rescheduling. Press this button to display the involved supply orders"

Reply:
Done!